community/values/src/main/java/org/neo4j/values/storable/NumberValue.java

```diff
@@ -137,4 +142,20 @@ else if ( numberValue instanceof FloatingPointValue )
             throw new IllegalArgumentException( "Cannot add " + numberValue );
         }
     }
+
+    public NumberValue divideBy( NumberValue numberValue )
+    {
+        if ( numberValue instanceof IntegralValue )
+        {
+            return dividedBy( numberValue.longValue() );
+        }
+        else if ( numberValue instanceof FloatingPointValue )
+        {
+            return dividedBy( numberValue.doubleValue() );
+        }
+        else
+        {
+            throw new IllegalArgumentException( "Cannot add " + numberValue );
```

Comment:
This error msg (and similar for `times` and `minus`) seem to have copy paste artifacts.

Reply:
chocking 😱